common/beerocks/hostapd/CMakeLists.txt

```
@@ -0,0 +1,135 @@
+project(prplmseh_hostapd VERSION ${prplmesh_VERSION})
```

(nitpick) I prefer if the module name is the same as the directory name.

---

Also, careful with the spelling
```suggestion
project(prplmesh_hostapd VERSION ${prplmesh_VERSION})
```

---

@fvbogaert  - fixed once I searched for the test application and couldn't find it by name :smile:.
@arnout - I thought about it and came to the conclusion that `hostapd` would be _extremely_ confusing on one hand, and naming the directory `prplmesh_hostapd` would be odd. I couldn't end up with something good, so I am open for better naming.